lib/Doctrine/DBAL/Migrations/Version.php

```diff
@@ -403,11 +403,10 @@ private function executeRegisteredSql($dryRun = false, $timeAllQueries = false)
                 foreach ($this->sql as $key => $query) {
                     $queryStart = microtime(true);
 
+                    $this->outputSqlQuery($key, $query);
```

Comment:
This will probably break migrations that use `\PDO::*` or `\Doctrine\DBAL\Connection::*` parameter types (see #528)

Reply:
@mikeSimonson is this a problem?

Reply:
This should have been fixed by #560. I will try to add a test for that specific case.